src/ui.text-files/UFiles.vue

```diff
@@ -57,11 +64,11 @@ function onRemoveFile(fileId) {
       <slot>
         <UFile
           v-for="(file, index) in formattedFileList"
-          :id="file.id"
-          :key="file.id"
-          :label="file.label"
-          :url="file.url"
-          :image-url="file.imageUrl"
+          :id="file?.id"
+          :key="file?.id"
+          :label="file?.label"
+          :url="file?.url"
+          :image-url="file?.imageUrl"
```

Comment:
Replace UFileItem with File and you probably don't need those optional chaining operators